pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -168,6 +169,13 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	if err != nil {
 		return reconcile.Result{}, errors.Wrapf(err, "fails to create InferenceServicesConfig")
 	}
+
+	// Reconcile cabundleConfigMap
+	caBundleConfigMapReconciler := cabundleconfigmap.NewCaBundleConfigMapReconciler(r.Client, r.Scheme)
```

Comment:
@danielezonca could you please take a look at this comment?

Reply:
I think this approach is coherent with the design of KServe so I would leave it a is: the possibility to create a global configuration requires access to the namespace where the controller is so it is already a "protected" action.

As part of this other ticket to review the cluster scoped serving runtime we might consider to review also this option with some opt-in (or opt-out) flag so that the is no _cluster global_ configurations
- https://github.com/kserve/kserve/issues/3154

But for this PR I would keep current design

Reply:
thx for the feedback. I will resolve this.